jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -311,15 +311,15 @@ private object Watches {
   def buildGroups(groups: Seq[Int]): Seq[Int] = {
```

Comment:
good to know that iterator.toSeq actually returns a Stream!

thanks, will merge once Travis is done